src/main/java/com/example/codeE/mapper/user/UserFromExcel.java

```diff
@@ -0,0 +1,40 @@
+package com.example.codeE.mapper.user;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Id;
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Pattern;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class UserFromExcel {
+    @Id
```

Comment:
Careful of this ID. In the DB, ID is an incremental value, this is maybe undesired.

Reply:
An entity class must declare at least 1 '@Id' or '@Embedded' property